apps/tmbitwarden.sh

```diff
@@ -79,38 +82,80 @@ Basically:
 Please have a look at how the questions are answered here if you are uncertain:
 https://i.imgur.com/YPynDAf.png"
 
+# Set a specific user for Bitwarden
+#print_text_in_color "$ICyan" "Specifying a ceratin user for Bitwarden: $BITWARDEN_USER..."
+#adduser --disabled-password --gecos "" "$BITWARDEN_USER"
+#sudo usermod -aG docker "$BITWARDEN_USER"
+#usermod -s /bin/bash "$BITWARDEN_USER"
+
+# Create bitwarden user and service
+if ! id "$BITWARDEN_USER" >/dev/null 2>&1
+then
+    useradd -r "$BITWARDEN_USER"
+fi
+groupadd docker >/dev/null 2>&1
+sudo usermod -aG docker "$BITWARDEN_USER"
+if [ ! -d /home/"$BITWARDEN_USER" ]
+then
+    mkdir -p /home/"$BITWARDEN_USER"
+    chmod -R "$BITWARDEN_USER"n:docker /home/"$BITWARDEN_USER"
+fi
+
+# Create the service
+cat << BITWARDEN_SERVICE > /etc/systemd/system/bitwarden.service
+[Unit]
+Description=Bitwarden
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=oneshot
+User="$BITWARDEN_USER"
+Group="$BITWARDEN_USER"
+ExecStart=/home/"$BITWARDEN_USER"/bitwarden.sh start
+RemainAfterExit=true
+ExecStop=/home/"$BITWARDEN_USER"/bitwarden.sh stop
+
+[Install]
+WantedBy=multi-user.target
+BITWARDEN_SERVICE
+
+sudo chmod 644 /etc/systemd/system/bitwarden.service
+systemctl daemon-reload
+sudo systemctl enable bitwarden.service
```

Comment:
btw: this cannot work here since we have not installed bitwarden here, yet

Reply:
Please move it. Thanks!

Reply:
done